packages/framework-core/src/services/event-store.ts

```diff
@@ -73,6 +73,18 @@ export class EventStore {
     }
   }
 
+  @Trace(TraceActionTypes.CUSTOM)
+  public async storeDispatchedEvent(eventEnvelope: EventEnvelope) {
```

Comment:
Not sure how this passes lint, but you missed the return type `Promise<boolean>`